### ISSUE TITLE
Respect the * option to ref in hyperref

### ISSUE THREAD

**Issue by teepeemm** [brucemiller/LaTeXML#1710]
Could we adjust `\ref` and `\pageref` so that they respect the `*` options provided by hyperref?  Looking at `autoref`, I think that means that we would need to change  
`"<ltx:ref labelref='#label' _force_font='true'/>"`  
to become  
`"<ltx:ref ?#1(class='ltx_nolink')() labelref='#label' _force_font='true'/>"`  
at line 3650 and 3652 of LaTeX.pool.ltxml.

A related advantage is that this would allow `\[\text{\ref*{label}}\]`, which is currently breaking MathJax as in #1477 and #1708 (which is their fault, but at least this would let us work around it).

**Comment by teepeemm** [comment]
It appears that the related advantage doesn't happen: it still becomes
```xml
<mtext class="ltx_refmacro_autoref ltx_nolink">
<span class="ltx_ref ltx_refmacro_autoref ltx_nolink ltx_ref_self">
<span class="ltx_text ltx_ref_tag">Equation 1</span>
</span>
 </mtext>
```
which MathJax can't handle.  Would we also want to adjust the xslt to drop the spans if there's nothing else in the mtext?  Or not work around a MathJax bug that much?

**Comment by teepeemm** [comment]
This was resolved by pr #1860.
